tests/examples/recipe_nonWF.slim

```diff
@@ -1,8 +1,8 @@
 initialize()
 {
-    setSeed(23);
-    defineConstant("OUTFILE", "recipe_nonWF.trees");
-    defineConstant("PEDIGREE", paste0(c(OUTFILE, ".pedigree")));
+    // OUTPREFIX can be defined on the command line to save in a different dir
```

Comment:
whoops!

Reply:
hm, and all the other recipes have the `setSeed(23)` line (which I think is a good idea) - should put that back in

Reply:
Oops. Slippery fingers. Will reinstate.